### ISSUE TITLE
Not is possible translate the call's direction neither task's priority

### ISSUE THREAD

**Issue by tomferreira** [oroinc/crm#290]
I'd like translate the call's direction and task's priority, but this is yet not possible.

**Comment by mbessolov** [comment]
You can  do it in System -> Entity -> Entity Management by switching your current language - when you saving the entity or entity field the labels you entered instead of the default ones will be associated with the language that you  currently have selected for your user session.

**Comment by tomferreira** [comment]
Yes, I know.. but in this case it's not possible change this values, because them are fixed in source code.

See this case:
https://github.com/oroinc/OroCRMTaskBundle/blob/90d55ca4f0a3cd9d1dc95c0b810d58f710dec991/Migrations/Data/ORM/LoadTaskPriority.php#L15

Thanks for quick response.
